### PR TITLE
Refactored same code in a class

### PR DESCRIPTION
**Pull request by aashishdhawan**
Refactoring the common function into one place.

### CODE REVIEW

File: Source/CameraView/CameraView.swift [hunk 1]
```diff
@@ -9,7 +9,7 @@ protocol CameraViewDelegate: class {
 }
 
 class CameraView: UIViewController, CLLocationManagerDelegate {
-
+
```
Comment (zenangst) [inline Source/CameraView/CameraView.swift]: @aashishdhawan What is going on here? 🤔
We have an Xcode plugin installed to remove such white spaces on save, you might want to try it - https://github.com/zenangst/MarvinXcode

**Comment by zenangst** [comment]
@aashishdhawan this is great! ^^

**Comment by zenangst** [comment]
Oh, now we have some merge conflicts, mind fixing them @aashishdhawan ?

**Comment by aashishdhawan** [comment]
@zenangst  Ready for another review.

**Comment by RamonGilabert** [comment]
Yo this is pretty great! Thank you for contributing! :) :sushi:

**Comment by zenangst** [comment]
@aashishdhawan 

unknown-9

**Pull request merged**